backend/app/api/api_v1/schemas/document.py

```diff
@@ -127,4 +128,17 @@ class RelationshipAndDocumentsGetResponse(BaseModel):
     """Response for Relationship get request."""
 
     relationship: RelationshipEntityResponse
-    documents: Optional[List[DocumentOverviewResponse]] = None
+    documents: Optional[Sequence[DocumentOverviewResponse]] = None
+
+
+class DocumentUploadCompleteRequest(BaseModel):
+    """Information generated during the upload of a document that should be stored."""
+
+    md5_sum: str
+    content_type: str
+
+
+class BulkImportResult(BaseModel):
```

Comment:
Without going to full on Java ... I was thinking of naming this `BulkImportValidatedResult` -  to signify that the request has been successfully validated as well as initiated?

Reply:
agreed, we only return this on success, so this is a validated count now (and maybe more later)

Reply:
Great!